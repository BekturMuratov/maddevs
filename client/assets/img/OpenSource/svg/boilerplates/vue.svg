<svg xmlns="http://www.w3.org/2000/svg" width="336" height="125" fill="none" xmlns:v="https://vecta.io/nano"><g clip-path="url(#A)"><path d="M206.784.001l-38.797 67.203L129.191.001H-.008l167.996 290.997L335.984.001h-129.2z" fill="#41b883"/><path d="M206.785.001l-38.797 67.203L129.191.001h-62l100.798 174.595L268.786.001h-62.001z" fill="#34495e"/></g><defs><clipPath id="A"><path fill="#fff" d="M0 0h336v291H0z"/></clipPath></defs></svg>