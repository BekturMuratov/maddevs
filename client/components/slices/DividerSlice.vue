<template>
  <hr>
</template>

<script>
export default {
  name: 'Divider',
}
</script>
