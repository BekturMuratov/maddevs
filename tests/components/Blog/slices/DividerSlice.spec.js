import DividerSlice from '@/components/Blog/slices/DividerSlice'
import { render } from '@testing-library/vue'

describe('DividerSlice component', () => {
  it('should render correctly', () => {
    const { container } = render(DividerSlice)

    expect(container).toMatchSnapshot()
  })
})
