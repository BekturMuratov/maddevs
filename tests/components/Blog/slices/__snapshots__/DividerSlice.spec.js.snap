// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DividerSlice component should render correctly 1`] = `
<div>
  <hr />
</div>
`;
